spark/src/main/scala/geotrellis/spark/tiling/ZoomedLayoutScheme.scala

```diff
@@ -116,13 +116,15 @@ class ZoomedLayoutScheme(val crs: CRS, val tileSize: Int, resolutionThreshold: D
 
   def zoomOut(level: LayoutLevel) = {
     val layout = level.layout
+    val newZoom = level.zoom - 1
+    val newSize = math.pow(2, newZoom).toInt
```

Comment:
@echeipesh what do you mean by that? as floating layout scheme has no `def levelFor(extent: Extent, cellSize: CellSize)` function, I am not sure how to identify max zoom level for the floating layout scheme